tests/x11/evolution/evolution_prepare_servers.pm

```diff
@@ -51,7 +52,8 @@ sub run() {
     assert_script_run "openssl dhparam -out /etc/dovecot/dh.pem 2048", 300;
 
     # Generate default certificate for dovecot and postfix
-    assert_script_run "cd /usr/share/doc/packages/dovecot;bash mkcert.sh";
+    my $dovecot_path = is_jeos() ? '/usr/share/dovecot' : '/usr/share/doc/packages/dovecot';
```

Comment:
Not sure that `()` is really needed (but maybe tidy complains about it?), but that's more cosmetic :)

Reply:
I had it without () and the CI test failed because of it.

Reply:
Ok, I was not sure, something learned!